website/__init__.py

```diff
@@ -1,11 +1,13 @@
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from flask_login import login_manager
 
 db = SQLAlchemy()
 
 def create_app():
     app = Flask(__name__)
-    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:abc@localhost:5432/shop_easy'
+    app.config['SECRET_KEY'] = 'secretkey'
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://postgres:password@localhost:5432/shop_easy'
```

Comment:
'password' is your password for PostgreSQL?

Reply:
Yes, It is.